bliss/datasets/galsim_galaxies.py

```diff
@@ -229,9 +229,9 @@ def __init__(
         assert sdss_kwargs["bands"][0] == 2
         assert len(list(psf_points)) == 2
         sdss_data = SloanDigitalSkySurvey(**sdss_kwargs)
-        _psf = sdss_data.rcfgcs[0][-1]
```

Comment:
The convention is that a variable beginning with an underscore will not be used. But for some reason lots of variables in our codebase start with underscores even though they are used. Really confusing.